package.json

```diff
@@ -17,14 +17,28 @@
     "hoek": "2.x.x"
   },
   "devDependencies": {
+    "autoprefixer-stylus": "0.8.0",
```

Comment:
The consistency is:
- hapi/cl modules are `#.x.x`
- outside modules version > 1 => `#.#.x`
- outside modules version < 1 => `#.#.#`